distutils/sysconfig.py

```diff
@@ -142,13 +145,17 @@ def get_python_lib(plat_specific=0, standard_lib=0, prefix=None):
     If 'prefix' is supplied, use it instead of sys.base_prefix or
     sys.base_exec_prefix -- i.e., ignore 'plat_specific'.
     """
+    
+    implementation = 'python'
     if IS_PYPY:
-        # PyPy-specific schema
-        if prefix is None:
-            prefix = PREFIX
-        if standard_lib:
-            return os.path.join(prefix, "lib-python", sys.version[0])
-        return os.path.join(prefix, 'site-packages')
+        if sys.version_info < (3, 8):
+            # PyPy-specific schema
+            if prefix is None:
+                prefix = PREFIX
+            if standard_lib:
+                return os.path.join(prefix, "lib-python", sys.version[0])
+            return os.path.join(prefix, 'site-packages')
+        implementation = 'pypy'
```

Comment:
Same here:

```python
implementation = 'pypy' if IS_PYPY else 'python'
if IS_PYPY and sys.version_info < (3, 8):
    # PyPy-specific schema
    if prefix is None:
        prefix = PREFIX
    if standard_lib:
        return os.path.join(prefix, "lib-python", sys.version[0])
    return os.path.join(prefix, 'site-packages')
```

(github does not let me make code suggestions in this location)

Reply:
adopted